lib/cf/CloudControllerClient.js

```diff
@@ -275,6 +278,27 @@ class CloudControllerClient extends HttpClient {
     return this.getResource(`/v2/organizations/${org_guid}`, options);
   }
 
+  getMultiTenantRabbitMQServiceInstanceIdWithName(rabbitmq_instance_name, org_guid, space_guid, options) {
```

Comment:
Is it fine if I make the signatures of the functions as?
getMultiTenantServiceInstanceIdWithName(instance_name, org_guid, space_guid, service_name, options)

Then instead of rabbitmq being in the name of the function I will pass it as a parameter from the VirtualHostManager which is anyways very specific to rabbitmq. The method inside CloudControllerClient will become generic because all the references to rabbitmq will be gone.

Reply:
Yes that is better. Can you please patch it? Let's see how it looks like.